### ISSUE TITLE
target-determinator does not evaluate aliases as changed

### ISSUE THREAD

**Issue by mark-thm** [bazel-contrib/target-determinator#63]
In my project I have a setup akin to:

```
py_library(
  name = "foo"
  ...
)

alias(name = "bar", actual = ":foo")
```

When I've made changes that invalidate foo and running:
```
target-determinator -targets //foo HEAD~1
```
I observe, correctly that `//foo` has been identified as a changed target.

However, for the _same_ changeset that invalidates foo and running:
```
target-determinator -targets //bar HEAD~1
```
`//bar` is not listed as changed.

**Comment by illicitonion** [comment]
Good find, this is a weird intersection of Bazel bugs! https://github.com/bazel-contrib/target-determinator/pull/64 should fix it up.

**Comment by illicitonion** [comment]
The fix for this was just released in v0.20.0 - thanks again for the bug report!
